test/parallel/test-stdin-require.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const execSync = require('child_process').execSync;
+const fs = require('fs');
+const path = require('path');
+
+if (common.isWindows) {
+  common.skip('Skipping test on windows');
```

Comment:
Yea, as of right now, I just don't know how to reproduce on windows and I don't have access to a windows box.